precompile/modules/initia_stdlib/sources/vip/vip.move

```diff
@@ -290,6 +346,24 @@ module initia_std::vip {
         target_hash
     }
 
+    fun make_challenge_table_key(
+        execution_time: u64,
+        stage: u64,
+        bridge_id: u64
+    ): vector<u8> {
+        // make key of executed_challenge
+        let encode: vector<u8> = bcs::to_bytes<u64>(&execution_time);
```

Comment:
default action of bcs is little endian, so if we want to use key sort feature we need to use table_key::encode_u64